### ISSUE TITLE
how to understand the GAP code?

### ISSUE THREAD

**Issue by shijh1975** [sthalles/deeplab_v3#11]
image_level_features = tf.reduce_mean(net, [1, 2], name='image_level_global_pool', keep_dims=True)
 image_level_features = slim.conv2d(image_level_features, depth, [1, 1], scope="image_level_conv_1x1",
                                           activation_fn=None)
 image_level_features = tf.image.resize_bilinear(image_level_features, (feature_map_size[1],                   feature_map_size[2]))
 The dim of input tensor net is [ batch,height,width,deepth], so after resuce_mean it became to [batch,1,1,deepth], afetr 1X1 conv, it became to[batch,1,1,256], I want to know why it can to be[batch,heihgt,width,256] after resize?

**Comment by sthalles** [comment]
Please, refer to the blog post.
